nfs-ganesha/build/build_rpm

```diff
@@ -41,12 +41,12 @@ cmake -DCMAKE_BUILDER_TYPE=Maintainer -DUSE_FSAL_RGW=YES -DUSE_FSAL_CEPH=YES $WO
 ## Create the source rpm
 echo "Building SRPM"
 rpmbuild \
-    --define "_sourcedir ./dist" \
+    --define "_sourcedir $WORKSPACE/dist" \
     --define "_specdir ." \
```

Comment:
@b-ranto, do i need to change the paths for _specdir, _builddir, _scrpmdir, and _rpmdir to $WORKSPACE/dist in this command?